src/cf-forms/usage.md

```diff
@@ -75,7 +76,6 @@ Theme variables for setting the color and sizes throughout the project. Overwrit
 </label>
 ```
 
-
```

Comment:
Trivial: I think two blank lines above an `##` is our unwritten standard. I would like it to be, anyway ;)